charts/redpanda/templates/_helpers.tpl

```diff
@@ -351,9 +346,36 @@ Generate configuration needed for rpk
 {{ join " " (list $flags.brokers $flags.admin $flags.sasl $flags.kafka)}}
 {{- end -}}
 
+{{- define "rpk-common-flags-no-admin" -}}
```

Comment:
These aren't really common to anything else, so let's just drop the common part.

```suggestion
{{- define "rpk-flags-no-admin" -}}
```